src/routes/Cognitive/Colors/Colors.scss

```diff
@@ -0,0 +1,49 @@
+.colors {
+  &__square {
+    position: relative;
+    width: 100%;
+    border: grey 1px solid;
```

Comment:
Hmm why this border?

Reply:
Cuz if the color is white, may not be obvious that there is an option. Hmm do you have any suggestion on the UX part? 

![Screenshot from 2020-05-18 19-24-00](https://user-images.githubusercontent.com/36331541/82207898-2883c300-993d-11ea-8a1f-22107ca4d647.png)